web_app/calendar/app.py

```diff
@@ -234,10 +244,11 @@ def daily_page():
     set_parameters(base_url=request.url)
 
     archive = True if request.args.get('archive') == 'True' else False
-
-    return daily_body.format(table=gen_daily_cells(user_id, archive),
+    body = daily_body.format(table=gen_daily_cells(user_id, archive),
                              archive=not archive)
 
+    return body.replace('>Daily<', '>Archive<') if archive else body
```

Comment:
Please, replace 'Daily' without any symbols. If you have multiple occurrences, better use regular expressions.